sort.rb

```diff
@@ -0,0 +1,25 @@
+
+def sort(array) 
+  if array == nil || array.length <= 1
+    return array # nothing to sort
+  end
+
+  index = 0
+  swapped = true
+  while index < array.length && swapped
+    swapped = false
+    other_index = index + 1
```

Comment:
is there a better variable name for other_index?

Reply:
Again, nice call.

Reply:
good catch!